docs/architecture/components.md

```diff
@@ -0,0 +1,9 @@
+# Components
+
+## 1. Offline VC sharing component
```

Comment:
Is the number required in the above line?

Reply:
I am thinking face auth sdk will come as another component. Irrespective, we can remove it as well